src/network.h

```diff
@@ -0,0 +1,22 @@
+#ifndef network_h
+#define network_h
+#include <arpa/inet.h>
+#include <netdb.h>
+#include <netinet/in.h>
+#include <sys/select.h>
+#include <sys/types.h>
+
+#include "canvas.h"
+#include "view.h"
+
+typedef struct NET_CFG {
+  fd_set clientfds;
+  int sockfd;
+} Net_cfg;
+
+Canvas *net_init(char *hostname, char *port);
+Net_cfg *net_getcfg();
+void net_handler(View *view);
+void net_send_char(int y, int x, char ch);
+
+#endif
```

Comment:
```suggestion
#endif

```
`"files.insertFinalNewline": true` in vscode `config.json`